eng/Signing.props

```diff
@@ -42,6 +42,7 @@
     <FileSignInfo Include="Nerdbank.Streams.dll" CertificateName="3PartySHA2" />
     <FileSignInfo Include="Newtonsoft.Json.dll" CertificateName="3PartySHA2" />
     <FileSignInfo Include="Validation.dll" CertificateName="3PartySHA2" />
+    <FileSignInfo Include="Humanizer.dll" CertificateName="3PartySHA2" />
```

Comment:
Anyone more familiar with signing that could tell me if this is correct or not?  Without this, signing gave warnings that we were signing a 3rd party dll with the microsoft dotnet cert.
@dotnet/roslyn-infrastructure

Reply:
Cert looks good.